src/main/java/com/novoda/notils/exception/DeveloperError.java

```diff
@@ -12,7 +12,12 @@ public DeveloperError(String detailMessage) {
         super(detailMessage);
     }
 
+    public DeveloperError(String messageTemplate, String... args) {
```

Comment:
> If the argument arg is null, then the result is "null". If arg implements Formattable, then arg.formatTo is invoked. Otherwise, the result is obtained by invoking arg.toString().

:+1: @xrigau